ucsdlib/hifive
# Add Recognition email function

Fixes #22 

#### Local Checklist
- [x] Tests written and passing locally?
- [x] Code style checked?
- [x] QA-ed locally?
- [x] Rebased with `master` branch?
- [x] Configuration updated (if needed)?

#### What does this PR do?
Send a Recognition email upon submission/creation of a Recognition.

##### Why are we doing this? Any context of related work?
References #22 

#### Manual testing steps?
From development environment, create a new Recognition and submit it.
Visit http://192.168.99.100:3000/letter_opener to see the email.

#### Screenshots
![recognitionemail](https://user-images.githubusercontent.com/1864660/48293166-a4d94400-e432-11e8-9770-b310e3a77b07.png)

---

@mcritchlow  - please review
